package.json

```diff
@@ -4,14 +4,18 @@
   "module": "dist/index.mjs",
   "main": "dist/index.js",
   "scripts": {
+    "postinstall": "npm run build",
```

Comment:
To be more closely aligned with the current template I would remove the `postinstall`